openmrs-19/src/main/java/org/motechproject/openmrs19/resource/PatientResource.java

```diff
@@ -86,4 +98,14 @@
      * @param newMotechId  the new MOTECH Id
      */
     void updatePatientMotechId(Config config, String patientUuid, String newMotechId);
+
+    /**
+     * Updates the identifier for the patient with given UUID. The given {@code config} will be used while performing
+     * this action.
+     *
+     * @param config  the configuration to be used while performing this action
+     * @param patientUuid  the UUID of the patient
+     * @param updatedIdentifier  the updated patient identifier
+     */
+    void updatePatientIdentifier(Config config, String patientUuid, Identifier updatedIdentifier);
```

Comment:
change the name of variable to identifier and add info that it is an identifier to be updated